### PR TITLE
8252195: AWT Accessibility API nested classes rely on default constructors

### PR DESCRIPTION
**Pull request by prrace**
https://bugs.openjdk.java.net/browse/JDK-8252195
is another one of the issues adding missing explicit no-args constructors in the desktop module.

As well as being nested, these are all concrete, but protected, classes and so the constructors
are protected.

CSR here https://bugs.openjdk.java.net/browse/JDK-8253450 needs a reviewer too.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8252195](https://bugs.openjdk.java.net/browse/JDK-8252195): AWT Accessibility API nested classes rely on default constructors

### Reviewers
 * [Sergey Bylokhov](https://openjdk.java.net/census#serb) (@mrserb - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/288/head:pull/288`
`$ git checkout pull/288`

### CODE REVIEW

File: src/java.desktop/share/classes/java/LS [hunk 1]
```diff
@@ -0,0 +1,17 @@
+-rw-r--r--   1 prrace  staff   22640 Sep 21 12:12 CheckboxMenuItem.java
```
Comment (mrserb) [inline src/java.desktop/share/classes/java/LS:1]: This extra file seems accidentally added

**Comment by prrace** [comment]
extra file deleted. That'll teach me not to do 'git add <folder>'

**Review by mrserb — APPROVED** [review]

**Pull request closed without merging**